service/controller/v1/key/key.go

```diff
@@ -6,8 +6,13 @@ import (
 )
 
 const (
-	NginxICMetricPort   = "10254"
-	KubeStaeMetricsPort = "10301"
+	NginxICMetricPort       = "10254"
```

Comment:
Makes the names long but I think using `NginxIngressController` as a prefix would be clearer. WDYT?

Reply:
Much better, done.